File: Assignment 5/Q4.c
/* Write a program which accepts N from user and print all odd numbers up to N.
   Input : 18
   Output : 1   3   5   7   9   11  13  17
*/
#include<stdio.h>

void OddDisplay(int iNO)
{
    int iCount = 0;
    for(iCount = 1; iCount<=iNO; iCount++)
    {
        if((iCount%2) != 0)
        {
            printf("%d\t",iCount);
        }
    }
}

int main()
{
    int iValue = 0;

    printf("Enter Number : ");
    scanf("%d",&iValue);

    OddDisplay(iValue);

    return 0;
}

File: Assignment 17/Q4.java
// Write a Program to find the Minimum of three Numbers.

class Logic
{
    void findMin(int a, int b, int c)
    {
         if((a < b) && (a < c))
        {
            System.out.println("Minimum Number is : " + a);
        }
        else if((b < a) && (b < c))
        {
            System.out.println("Minimum Number is : " + b);
        }
        else
        {
            System.out.println("Minimum Number is : " + c);
        }
    }
}

class Q4
{
    public static void main(String args[])
    {
        Logic obj = new Logic();
        obj.findMin(3, 7, 2);
    }
}